{{ cookiecutter.repo_directory }}/index.html

```diff
@@ -2,6 +2,12 @@
 <html lang="en">
 
 <head>
+    {%- if 'redirect' in cookiecutter %}
+    <meta http-equiv="refresh" content="0; {{cookiecutter.redirect.url}}">
+    <script>
+         window.location.replace("{{cookiecutter.redirect.url}}");
+    </script>
```

Comment:
This is the "old school" Java script way, before HTML-5 was a widely adapted standard in the browser world.